whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -400,9 +400,13 @@ public interface AttributeSyntax extends Documented {
 
     AttributeSyntax ORG_NAME_SYNTAX = new AttributeSyntaxRegexp(
             Pattern.compile("(?i)^[\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}( [\\]\\[A-Z0-9._\"*()@,&:!'`+\\/-]{1,64}){0,29}$"), "" +
-            "A list of 1 to 30 words separated by white space. A word is made up of letters, digits and the following characters:\n" +
+            "A list of 1 to 30 words separated by white space\n." +
+            "It just allow ASCII alphanumeric characters.\n" +
```

Comment:
I suggest `A word may only contain ASCII characters.` Not just alphanumeric - punctuation is also allowed.